test-data/snake_oil_structure/ert/model/user_config.ert

```diff
@@ -58,7 +58,7 @@ GEN_DATA super_data INPUT_FORMAT:ASCII RESULT_FILE:super_data_%d  REPORT_STEPS:1
 -- ------ ---------------------------------------------------------------------------------------------------------------------
 -- ------ ert identifier    template file                          result file             parameter distribution
 -- ------ ---------------------------------------------------------------------------------------------------------------------
-GEN_KW SIGMA          ../input/templates/sigma.tmpl          coarse.sigma              ../input/distributions/sigma.dist
+GEN_KW SIGMA ../input/templates/sigma.tmpl coarse.sigma ../input/distributions/sigma.dist
```

Comment:
think you are messing up the alignment of the columns here by removing spaces ?

Reply:
yes, i am : D fixed it - it was a long line, and I've just added a pylint ignore instruction instead of messing with space.